### PR TITLE
system variable: rename tidb_ddl_distribute_reorg to tidb_enable_dist_task

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/13549
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1153,18 +1153,18 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
 
 - Scope: GLOBAL
 - Persists to cluster: Yes
 - Default value: `OFF`
-- This variable is used to control whether to enable distributed execution of the DDL reorg phase to improve the speed of this phase. Currently, this variable is only valid for the `ADD INDEX` statement. Enabling this variable improves the performance of large tables. Distributed DDL execution can control the CPU usage of DDL through dynamic DDL resource management to prevent DDL from affecting the online application.
-- To verify whether a completed `ADD INDEX` operation is accelerated by this feature, you can check whether a corresponding task is in the `mysql.tidb_background_subtask_history` table.
+- This variable is used to control whether to enable [the distributed execution framework](/tidb-distributed-execution-framework.md). After enabling distributed execution, DDL, Import and other supported backend tasks will be jointly completed by multiple TiDB nodes in the cluster.
```
Comment (ran-huang) [inline system-variables.md:1165]:
```suggestion
- This variable is used to control whether to enable [the distributed execution framework](/tidb-distributed-execution-framework.md). After enabling distributed execution, backend tasks such as DDL and import will be jointly completed by multiple TiDB nodes in the cluster.
```

File: system-variables.md [hunk 2]
```diff
@@ -1153,18 +1153,18 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
+- In TiDB v7.1.0, the framework supports distributedly executing the `ADD INDEX` statement for partitioned tables.
```
Comment (ran-huang) [inline system-variables.md:1166]:
```suggestion
- In TiDB v7.1.0, the framework supports distributedly executing only the `ADD INDEX` statement for partitioned tables.
```

File: system-variables.md [hunk 3]
```diff
@@ -1153,18 +1153,18 @@ MPP is a distributed computing framework provided by the TiFlash engine, which a
...
 
 - Scope: GLOBAL
 - Persists to cluster: Yes
 - Default value: `OFF`
-- This variable is used to control whether to enable distributed execution of the DDL reorg phase to improve the speed of this phase. Currently, this variable is only valid for the `ADD INDEX` statement. Enabling this variable improves the performance of large tables. Distributed DDL execution can control the CPU usage of DDL through dynamic DDL resource management to prevent DDL from affecting the online application.
-- To verify whether a completed `ADD INDEX` operation is accelerated by this feature, you can check whether a corresponding task is in the `mysql.tidb_background_subtask_history` table.
+- This variable is used to control whether to enable [the distributed execution framework](/tidb-distributed-execution-framework.md). After enabling distributed execution, backend tasks such as DDL and import will be jointly completed by multiple TiDB nodes in the cluster.
```
Comment (qiancai) [inline system-variables.md:1165]: As tidb-distributed-execution-framework.md will be ready next month, the reference to it can be removed for now.

Comment (qiancai) [inline system-variables.md:1165]:
```suggestion
- This variable is used to control whether to enable the distributed execution framework. After it is enabled, backend tasks such as DDL and import will be jointly completed by multiple TiDB nodes in the cluster.
```

**Review by qiancai — APPROVED** [review]
other LGTM

**Pull request merged**